### DISCUSSION TITLE
H2数据库数据迁移到mysql数据库

### DISCUSSION THREAD

**Discussion by neilsof** [halo-dev/halo#3283] [Q&A]
### 你当前使用的版本

2.1.0

### 描述一下此特性

当前可使用halo-next文件，通过H2的命令将数据导出为sql文件，但是遇到H2的data数据结构与mysql的data属性数据结构不同。
这边是否有工具脚本将H2的数据转换为mysql的数据结构。
感谢

### 附加信息

![image](https://user-images.githubusercontent.com/45843539/217197992-a6798208-424e-4f58-a562-1cd0869654f3.png)

![image](https://user-images.githubusercontent.com/45843539/217198074-d90a452c-1040-4492-a2cd-47357984eeb1.png)

**Comment by ruibaby** [comment]
可以尝试一下这个工具：https://dbcopyplugin.sourceforge.net/
